accounts/manager.go

```diff
@@ -29,10 +29,10 @@ import (
 	"sync"
 	"time"
 
+	"encoding/json"
```

Comment:
Just a minor subjective stylistic suggestion, we could separate standard library imports from custom imports so
```go
    "encoding/json"
    "path/filepath"
    "sync"
    "time"

    "github.com/ethereumproject/go-ethereum/common"
```
and after go-fmt, that way it'll be easy to isolate/identify packages